cmd/main.go

```diff
@@ -1,17 +1,32 @@
 package main
 
 import (
+	"fmt"
 	"os"
+	"os/signal"
+	"syscall"
 
 	"github.com/kyma-project/cli/cmd/kyma"
 	"github.com/kyma-project/cli/internal/cli"
 )
 
 func main() {
+	SetupCloseHandler()
 	command := kyma.NewCmd(cli.NewOptions())
 
 	err := command.Execute()
 	if err != nil {
 		os.Exit(1)
 	}
+
+}
+
+func SetupCloseHandler() {
+	c := make(chan os.Signal, 2)
+	signal.Notify(c, os.Interrupt, syscall.SIGINT)
```

Comment:
The signals `os.Interrupt` and `syscall.SIGINT` are the same, please remove one of them.
See golang's `os` snippet here:
https://golang.org/pkg/os/#Signal

Also, we should probably add `SIGTERM` as well to the signals we react on.